addons/pos_restaurant/tests/test_frontend.py

```diff
@@ -269,3 +269,8 @@ def test_10_save_last_preparation_changes(self):
         self.start_tour("/pos/ui?config_id=%d" % self.pos_config.id, 'SaveLastPreparationChangesTour', login="pos_admin")
         self.assertTrue(self.pos_config.current_session_id.order_ids.last_order_preparation_change, "There should be a last order preparation change")
         self.assertTrue("Coca" in self.pos_config.current_session_id.order_ids.last_order_preparation_change, "The last order preparation change should contain 'Coca'")
+
+    def test_11_qrcode_ticket(self):
```

Comment:
Could you rename this tour (and test) to reflect the fact that you are actually testing the bill screen in pos restaurant? Something like `BillScreenTour`.